tests/unit/general/bin-version.bats

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bats
```

Comment:
I think these tests would be more worth-while with mocks so that they actually showed versions in the output and not just the partials.